### PR TITLE
Add methods to set/remove replication rules

### PR DESCRIPTION
**Pull request by poornas**
Needed for https://github.com/minio/mc/pull/3335

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,16 +3,14 @@ module github.com/minio/minio-go/v7
 go 1.12
 
 require (
-	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/json-iterator/go v1.1.10
 	github.com/minio/md5-simd v1.1.0
+	github.com/minio/minio v0.0.0-20200731035804-2174a228351a
```
Comment (harshavardhana) [inline go.mod:9]: minio-go cannot depend on server at any point in time @poornas ?

Comment (harshavardhana) [reply go.mod:9]: perhaps missing ` go mod tidy?`

File: go.mod [hunk 2]
```diff
@@ -3,16 +3,23 @@ module github.com/minio/minio-go/v7
 go 1.12
 
 require (
-	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/json-iterator/go v1.1.10
+	github.com/klauspost/cpuid v1.3.1 // indirect
```
Comment (harshavardhana) [inline go.mod:8]: none of these are used by `minio-go` I am not able to see where is this coming from?

Comment (poornas) [reply go.mod:8]: it is coming from the md5-simd dependency

**Pull request merged**